data/campaigns/Under_the_Burning_Suns/utils/abilities.cfg

```diff
@@ -372,8 +351,8 @@
 #define ABILITY_TAILWIND X Y
     [dummy]
         id = tailwind_{X}
-        name = _ "tailwind"+{X}
-        description = _ "At the start of the turn this unit increases movement points of surrounding units by " + "{X}"
+        name = _ "tailwind +"+{X}
+        description = _ "At the start of the turn this unit increases movement points of surrounding units by +"+{X}
```

Comment:
We'll need to do something about the description, although tacking it on the end is slightly less bad than embedding it in the middle. Worst-case, we need a po comment letting translators know that "number of movement points" will be appended. It would be better though to do something like what was done with the song verse.